src/src/pages/capabilities/KafkaCluster/MessageContracts.js

```diff
@@ -228,14 +226,20 @@ export default function MessageContracts({
             >
               {shownContracts.map((contract) => (
                 <option key={contract.id} value={contract.id}>
-                  Version {contract.schemaVersion}
+                  {`Version${contract.schemaVersion}${
+                    contract.status !== "Provisioned"
```

Comment:
inside of MessageContracts.js we have an enum for the statuses. Use that instead
<img width="342" alt="image" src="https://github.com/dfds/selfservice-portal/assets/5738476/d5803097-e8ef-4a59-a551-072b02ba92ff">